test/t/test_openssl.py

```diff
@@ -2,7 +2,7 @@
 
 
 class TestOpenssl:
-    @pytest.mark.complete("openssl ")
+    @pytest.mark.complete("openssl ", require_cmd=True)
     def test_1(self, completion):
         assert completion
```

Comment:
Let's assert that we have some known/expected completions here, to gain some confidence that the parsing actually works.

Reply:
Added some sub commands from each section in the help